static/js/editor.js

```diff
@@ -10,9 +9,11 @@ var editorContent = new Quill('#editorContent', {
   }
 })
 
+title.setAttribute('contenteditable', true)
+
 document.querySelector('form[edit]').addEventListener('submit', event => {
   const form = event.target
-  form.title.value = editorTitle.getContents()
-  form.content.value = editorContent.getContents()
+  form.title.value = title.innerHTML
```

Comment:
`form.title.value = title.textContent` I guess, as we don't want to allow HTML in the title. (we should even clean it BTW, but later I guess)